website/events/api/v2/admin/views.py

```diff
@@ -0,0 +1,143 @@
+import json
+
+from django.http import HttpResponse, Http404
+from oauth2_provider.contrib.rest_framework import IsAuthenticatedOrTokenHasScope
+from rest_framework import status
+from rest_framework.generics import get_object_or_404
+from rest_framework.views import APIView
+from rest_framework import filters as framework_filters
+
+from events import services
+from events.api.v2.admin import filters
+from events.api.v2.admin.permissions import IsOrganiser
+from events.api.v2.admin.serializers.event import EventListSerializer, EventSerializer
+from events.api.v2.admin.serializers.event_registration import (
+    EventRegistrationSerializer,
+)
+from events.models import Event, EventRegistration
+from thaliawebsite.api.v2.admin.views import (
+    AdminListAPIView,
+    AdminRetrieveAPIView,
+    AdminCreateAPIView,
+    AdminUpdateAPIView,
+    AdminDestroyAPIView,
+)
+import events.api.v2.filters as normal_filters
+
+
+class EventAdminListAPIView(AdminListAPIView):
```

Comment:
This API view triggers 205 SQL queries on my db. A prefetch on the activemembergroup would be nice

Reply:
goddamn good idea

![image](https://user-images.githubusercontent.com/1799914/119895673-592e0d80-bf3e-11eb-89eb-1c02e5339d1d.png)
![image](https://user-images.githubusercontent.com/1799914/119895688-5d5a2b00-bf3e-11eb-9202-ba317339ff47.png)